README.md

```diff
@@ -2,13 +2,13 @@
 
 Uniquely and automatically colors each VSCode window.
 
-<img src="https://raw.githubusercontent.com/stuartcrobinson/unique-window-colors/master/darkExample.png" alt="drawing" width="200"/> &nbsp;&nbsp;&nbsp;
```

Comment:
"Image URLs in README.md and CHANGELOG.md need to resolve to https URLs."

https://code.visualstudio.com/docs/extensions/publish-extension#_publishing-extensions

Reply:
this is so they can display on the marketplace page